experiments/AMIP/components/ocean/prescr_seaice_init.jl

```diff
@@ -78,8 +78,10 @@ function ice_rhs!(du, u, p, _)
     rhs = @. (-F_turb_energy - F_radiative + F_conductive) / (params.h * params.ρ * params.c)
 
     # do not count tendencies that lead to temperatures above freezing, and mask out no-ice areas
-    area_mask = Regridder.binary_mask.(area_fraction, threshold = eps(FT))
-    unphysical = @. Regridder.binary_mask.(T_freeze - (Y.T_sfc + FT(rhs) * FT(p.dt)), threshold = FT(0)) .* area_mask
+    area_threshold = eps(FT)
+    area_mask = Regridder.binary_mask.(area_fraction, area_threshold)
+    calc_threshold = FT(0)
```

Comment:
Why 0 instead of eps here?

Reply:
Because for the `area_mask`, the `area_fraction` we're evaluating could be very slightly positive and still regarded to be ice free, hence `eps(FT)`. For `unphysical`, this is a crude guess as to whether the `Y.T_sfc` will reach below the freezing point or not, so FT(0) is more accurate.